modules/base/encoding.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2014 The Gogs Authors. All rights reserved.
```

Comment:
How about create a new package named encoding?

Reply:
Or `charset`, `transform`

Reply:
`charset` seems more appropriate. I'll do that.